test/e2e/snaps/test-snap-namelookup.spec.js

```diff
@@ -0,0 +1,95 @@
+const {
```

Comment:
This test should only be tested in Flask: https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/run-all.js#L11-L14

Reply:
Added it in the latest commit. Thanks!